<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="map">
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>48</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>48</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>48</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>48</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionEmplace"/>
   <addaction name="actionEmplaceRect"/>
   <addaction name="actionEmplaceFill"/>
   <addaction name="separator"/>
   <addaction name="actionBackground"/>
   <addaction name="actionForeground"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEmplace">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Emplace</string>
   </property>
  </action>
  <action name="actionEmplaceRect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>EmplaceRect</string>
   </property>
  </action>
  <action name="actionEmplaceFill">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>EmplaceFill</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionBackground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Background</string>
   </property>
  </action>
  <action name="actionForeground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Foreground</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
